sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -66,6 +66,12 @@ object StaticSQLConf {
       .checkValue(cacheSize => cacheSize >= 0, "The maximum size of the cache must not be negative")
       .createWithDefault(1000)
 
+  val CODEGEN_CACHE_SIZE = buildStaticConf("spark.sql.codegen.cacheSize")
+      .doc("The maximum cache size for generated classes.")
```

Comment:
We need to explain what is the cache is. It is hard to tell based on this description.

Reply:
spark.sql.codegen.cache.maxEntries?